source/speechDictHandler/__init__.py

```diff
@@ -118,6 +117,15 @@ def sub(self, text):
 def processText(text):
 	if not globalVars.speechDictionaryProcessing:
 		return text
+	# #14689: API level < 2 UIA consoles have many blank lines, which slows processing to a halt
+	# Because from UIAHandler.constants import WinConsoleAPILevel
+	# cause crashes as global import, and probably slowdowns here
+	# use 2 instead of WinConsoleAPILevel.FORMATTED constant
+	focus = api.getFocusObject()
+	if hasattr(focus, "apiLevel") and focus.apiLevel < 2:

```

Comment:
Check for an instance of `WinConsoleUIA`.

Reply:
Like for WinConsoleAPILevel, apparently I should import WinConsoleUIA inside processText function to avoid crashes. And I'm afraid this could affect performance (but I accept denials, of course).

Reply:
Importing it inside the function should not effect performance